client/src/views/test.vue

```diff
@@ -0,0 +1,21 @@
+<template>
```

Comment:
Please remove this file from the commit.

Reply:
Will do! If I push my changes to this branch, it will be reflected in this pull request?

Reply:
Yes